src/main/kotlin/carracing/game/controller/RaceController.kt

```diff
@@ -0,0 +1,25 @@
+package carracing.game.controller
+
+import carracing.game.domain.data.Race
+import carracing.game.view.CarRacingView
+
+class RaceController(
+    private val race: Race,
+    private val carRacingView: CarRacingView = CarRacingView(),
+) {
+    fun startRace() {
+        generateRaceProgress()
+        showWinners()
+    }
+
+    private fun generateRaceProgress() {
+        race.generateRaceSequence().forEach {
+            carRacingView.printCurrentRaceState(it)
+        }
```

Comment:
Can we change it so that the results are displayed after the car racing game finishes? What would be the advantage of doing that? You can think of this in terms of web programming.